scripts/utils/installers/node-tools.sh

```
@@ -0,0 +1,331 @@
+		# Install html-validate via bun (offline HTML validation)
+		echo -e "${BLUE}Installing html-validate...${NC}"
+
+		# Ensure bun is available (should already be installed for prettier)
+		if ! ensure_bun_installed; then
+			exit 1
+		fi
+
+		# Read html-validate version from _tool_versions.py (single source of truth)
+		HTML_VALIDATE_VERSION=$(get_tool_version "html-validate") || exit 1
+
+		if [ $DRY_RUN -eq 1 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)